descriptors/ModuleDescriptor-template.json

```diff
@@ -68,7 +68,8 @@
           ],
           "pathPattern": "/consortia/{consortiumId}/tenants",
           "permissionsRequired": [
-            "consortia.tenants.item.post"
+            "consortia.tenants.item.post",
+            "consortia.consortia-configuration.item.post"
```

Comment:
not permissionsRequired, modulePermissions

Reply:
Ok, fixed